src/App.tsx

```diff
@@ -1,26 +1,146 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFooter } from './Components/Footer';
+import { TodoHeader } from './Components/Header';
+import { TodoMain } from './Components/Main';
+import { ErrorNotifications } from './Components/Errors';
+import { Filter } from './types/Filter';
 
-const USER_ID = 0;
+function filterTodos(list: Todo[], filterWord: Filter) {
+  switch (filterWord) {
+    case Filter.All:
+      return list;
+    case Filter.Active:
+      return list.filter(todo => !todo.completed);
+    case Filter.Completed:
+      return list.filter(todo => todo.completed);
+    default:
+      return list;
+  }
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(Filter.All);
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
use `enum` for all the errors
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorType>(ErrorType.Default);
```